src/dev/build/tasks/fetch_agent_versions_list.ts

```diff
@@ -22,6 +22,8 @@ const getAvailableVersions = async (log: ToolingLog) => {
   const url = 'https://www.elastic.co/content/product_versions';
   log.info('Fetching Elastic Agent versions list');
 
+  log.info('process.env.BUILDKITE_PULL_REQUEST: ' + process.env.BUILDKITE_PULL_REQUEST);
```

Comment:
This logs out a build number, so the condition should be good.

`info process.env.BUILDKITE_PULL_REQUEST: 154808`

Reply:
Yep looks good to me!